portal-api/src/scheduler/jobs/send-report-due-emails-job/index.test.js

```diff
@@ -2,10 +2,9 @@ const { subDays } = require('date-fns');
 const { produce } = require('immer');
 const sendReportDueEmailsJob = require('.');
```

Comment:
I haven't come across `require('.')` before, how does it work?

Reply:
This is because the file I'm referring to is an `index.js` file. Similarly to how I can just refer to the directory name when importing an `index.js` file from another directory, I can just use `'.'` when referring to an `index.js` file in the same directory. This is the same as `'./index'`.